binder/src/test/java/io/grpc/binder/internal/PendingAuthListenerTest.java

```diff
@@ -0,0 +1,103 @@
+package io.grpc.binder.internal;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.Status;
+
+@RunWith(JUnit4.class)
+
+public final class PendingAuthListenerTest {
+
+  @Rule public final MockitoRule mocks = MockitoJUnit.rule();
+
+  @Mock ServerCallHandler<Object, Object> next;
+  @Mock ServerCall<Object, Object> call;
+  @Mock ServerCall.Listener<Object> delegate;
+  @Captor ArgumentCaptor<Status> statusCaptor;
+
+  private final Metadata headers = new Metadata();
+  private final PendingAuthListener<Object, Object> listener = new PendingAuthListener<>();;
```

Comment:
Extra semicolon. Can you run this through g4 fix or similar?

Reply:
Update. Also checked Android Studio, no further warnings open.